plugin/trino-phoenix5/src/main/java/io/trino/plugin/phoenix5/PhoenixMergeTableHandle.java

```diff
@@ -15,42 +15,75 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import io.trino.plugin.jdbc.JdbcColumnHandle;
 import io.trino.plugin.jdbc.JdbcTableHandle;
+import io.trino.spi.connector.ColumnHandle;
 import io.trino.spi.connector.ConnectorMergeTableHandle;
+import io.trino.spi.connector.ConnectorTableHandle;
+import io.trino.spi.predicate.TupleDomain;
 
-public record PhoenixMergeTableHandle(JdbcTableHandle tableHandle, PhoenixOutputTableHandle phoenixOutputTableHandle, JdbcColumnHandle mergeRowIdColumnHandle)
+import static java.util.Objects.requireNonNull;
+
+public record PhoenixMergeTableHandle(
+        JdbcTableHandle tableHandle,
+        PhoenixOutputTableHandle insertHandle,
+        PhoenixOutputTableHandle updateHandle,
+        PhoenixOutputTableHandle deleteHandle,
```

Comment:
`PhoenixOutputTableHandle`s have similar data in it. I wonder if it is not a problem when serializing it that a lot of JSON will be actually duplicated.